astronomer/airflow/version_check/update_checks.py

```diff
@@ -302,7 +302,7 @@ def dismiss(self, version):
 
             return self.response(200)
 
-    def register(self, app, options, first_registration):
+    def register(self, app, options):
```

Comment:
Change this to `def register(self, app, *args, **kwargs)` and `super().register(app, *args, **kwargs)` and it should work with v1 and v2.

Reply:
@ephraimbuddy ^^